dev/sparktestsupport/modules.py

```diff
@@ -188,6 +196,14 @@ def __hash__(self):
     ],
 )
 
+api = Module(
+    name="api",
+    dependencies=[utils, unsafe],
+    source_file_regexes=[
+        "sql/api/",
```

Comment:
So, is this missed since Apache Spark 3.5.0 until now?

Reply:
Yes ...

Reply:
Yes ...